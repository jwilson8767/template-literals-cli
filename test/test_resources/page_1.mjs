export default (config)=>`
<html>
<head> <title>test_1</title> </head>
<body>

${config.projects.map((project)=>`
    <div>
        <h2>${project.title}</h2>
        ${Object.entries(project.figures).map(([k,v ])=>`
          <img src="${v}" alt="${k}">
        `).join('')}
    </div>
`).join('')}

${/* Use env to switch between minified and unminified javascript files */ '' }
${config.env === 'prod' ? `
  <script src="dist/main.min.js"></script>
` : `
  <script type="module" src="src/main.js"></script>
`}

</body>
</html>
`;
