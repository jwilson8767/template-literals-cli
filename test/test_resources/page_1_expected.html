<html>
<head> <title>test_1</title> </head>
<body>


    <div>
        <h2>The Best Project</h2>

          <img src="/images/sales_1mo.png" alt="sales_1mo">

          <img src="/images/sales_3mo.png" alt="sales_3mo">

    </div>




  <script type="module" src="src/main.js"></script>


</body>
</html>
